scripts/get_version.mjs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import xml2js from "xml2js";
+import fs from "fs/promises";
+
+/*
+  Inputs:
+  => platform: the platform to get the current version for
+
+  Outputs:
+  => the MAJOR.MINOR.PATCH formatted version number for the given platform
+*/
+async function getVersion(platform) {
+  // xmljs can parse both plist and xml files
+  const parseFile = async filePath => await xml2js.parseStringPromise(await fs.readFile(filePath));
+  switch (platform) {
+    case "android":
+    case "browser":
+      const { widget } = await parseFile("config.xml");
+      return widget.$.version;
+    case "ios":
+    case "osx":
+      const { 
+        plist: { 
+          dict: 
+            [{ 
+              key: plistKeys, 
+              string: plistValues 
+          }] 
+        } 
+      } = await parseFile(`apple/xcode/${platform}/Outline/Outline-Info.plist`);
+      return plistValues[plistKeys.indexOf("CFBundleShortVersionString")];
+    case "windows":
+      return "1.7.0";
```

Comment:
So does this mean, in the future we need to manually bump up this version number to release Windows and Linux app?

Reply:
Yeah, for now. Before it was free floating and didn't even live in the code at all.

Reply:
@jyyi1 Yes. See the initial step in the new proposed flow: https://docs.google.com/document/d/1TNBED78PnxUH987ZRXEZ0G2E-8vk92V6UbHLWS3Iq8Y/edit?resourcekey=0-oWsq__Qiyvp4RnZsAsTzww#heading=h.j7o4n4avrdxk